tests/unittest/llmapi/test_llm_args.py

```
@@ -3985,3 +3985,40 @@ def test_deepseek_resolves_auto_to_python_on_nixl(self) -> None:
+    from transformers import PretrainedConfig
+
+    from tensorrt_llm._torch.pyexecutor.model_loader import ModelLoader
+
+    checkpoint_loader = MagicMock()
+    checkpoint_loader.load_config.return_value = SimpleNamespace(
+        pretrained_config=PretrainedConfig(architectures=[architecture]),
+        mm_encoder_only=False,
+        spec_config=None,
+    )
+    llm_args = TorchLlmArgs(
+        model="/tmp/dummy_model",
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/dummy_model"

(S108)